Adapt `Alert` styling (#2891)
vivid-planet/comet: .changeset/fifty-buses-hope.md

---
"@comet/admin": major
---

> Fix styling of `Alert` according to Comet DXP design

Reviewer: This is more of an adaptation to the updated design not really a fix imo. 

```suggestion
Adapt the styling of `Alert` to match the updated Comet design
```

Author: 3de29d58effcc691c280cfea2e79fd98dd4f23e0

> Remove styling for `text` variant of Buttons used in `Alert`.
> Use `variant="outlined"` instead.

Reviewer: ```suggestion
Remove styling for the `text` variant of buttons used in `Alert`.
Use buttons with the `outlined` variant instead to adhere to the Comet design guidelines. 
```

Author: 3de29d58effcc691c280cfea2e79fd98dd4f23e0

```diff
        <Alert
    // ...
    action={

> -       <Button variant="text" startIcon={<ArrowRight />}>
> -           Action Text
> -       </Button>
> +       <Button variant="outlined" startIcon={<ArrowRight />}>
> +           Action Text
> +       </Button>

Reviewer: Only the one line needs to be changed 🙂 

```suggestion
-       <Button variant="text" startIcon={<ArrowRight />}>
+       <Button variant="outlined" startIcon={<ArrowRight />}>
            Action Text
        </Button>
```

Author: 48e32d4b4686d5626e6904102f219870eb50d178
